client/src/components/brushableHistogram/index.js

```diff
@@ -289,7 +290,8 @@ class HistogramBrush extends React.Component {
       .call(
         d3
           .brushX()
-          .on("brush", this.onBrush(field, x.invert).bind(this))
+          .on("start", this.onBrush(field, x.invert, "start").bind(this))
+          .on("brush", this.onBrush(field, x.invert, "brush").bind(this))
```

Comment:
Requesting a comment here re: why all 3 are necessary

Reply:
will add.    TL;DR - needed for History.

The essence is that we want to know when the drag starts so we can push the Redux state onto the history stack.   We don't want to push new history on each brush/drag, but only on the first one.